joblib/parallel.py

```diff
@@ -94,23 +91,91 @@ def _register_dask():
 }
 
 
-def get_active_backend(prefer=None, require=None, verbose=0):
+# Sentinels for the default values of the Parallel constructor and
```

Comment:
I agree it's not ideal, but there's a fair amount of entanglement between the different objects in this file and it's hard to split them without creating cyclic imports. It would at least require a third file containing the common things. Maybe better to leave that for another PR ?

Reply:
agreed.